public/css/designer.css

```diff
@@ -2,16 +2,26 @@
   height: 100%;
 }
 
-.height-40 {
-  height: 40%;
+.column-fitting {
+  display: -webkit-box;
```

Comment:
I don't remember exact commands for flex boxes, but ensure me that these and following new styles are not webkit-only

Reply:
All the lines you see in the class column-fitting are generated from a .sass file which has only two lines. The lines I use are supported from IE11 and onwards, so we should be fine.